server/controllers/wip/questionsController.ts

```diff
@@ -64,8 +65,7 @@ export default class QuestionsController extends BaseController<FormWizard.Multi
       const { reportConfig } = res.locals
 
       for (const question of report.questions) {
-        // TODO: Remove QID-stripping logic once removed from API
-        const fieldName = stripQidPrefix(question.code)
+        const fieldName = questionFieldName(question)
```

Comment:
Wouldn't be less opaque to use `question.code` directly? It would also have the benefit that you wouldn't need `questionFieldName()` at all and just use `question.code` or `question.id` as necessary (again without the level of indirection).

Reply:
both question config and question instances have a `code` property – but they mean very different things so it's safer to not have to remember each time which is which and how in fact you construct a field name